glide.lock

```diff
@@ -1,14 +1,16 @@
-hash: d6093eb1a6d4c1eaa9f2ad996ebf07748b0b3a0413f4ab9b9435df1d255567a5
-updated: 2017-09-25T17:19:51.449837394-07:00
+hash: b75ed7f632cfd185b6a8f8bc037b09d32e292f908af6c1e257e444e17a3a39ef
```

Comment:
tip: I’ve found that it’s easier to rebase if I keep glide.lock updates in their own commit. Easy to remove the commit when I rebase, then recreate with `glide up`, instead of dealing with glide.lock merge conflicts. More relevant on a more active repository.

Reply:
Yep, agreed. Since this repo isn't really updated, this hasn't been an issue.